components/merkledb/src/access/extensions.rs

```diff
@@ -63,7 +64,7 @@ use crate::{
 /// let list = patch.get_list::<_, String>("list");
 /// assert_eq!(list.len(), 1);
 /// ```
-pub trait AccessExt: Access {
+pub trait AccessExt: Access + Copy {
```

Comment:
Bikeshedding / opinions: I'd suggest to use the name `AccessExt` for a more generic extension trait (i.e., one which takes `&self` and doesn't require `Self: Copy`), and rename this trait to something like `CopyAccessExt`.